pkg/client/clientpool.go

```diff
@@ -71,11 +75,13 @@ func (p *ClientPool) Close() {
 func (p *ClientPool) Init() error {
 	stmt := fmt.Sprintf("USE %s; UPDATE CONFIGS storage:wal_ttl=3600; UPDATE CONFIGS storage:rocksdb_column_family_options = { disable_auto_compactions = true };", p.space)
 	for i := 0; i < p.concurrency; i++ {
-		if resp, err := p.Conns[i].Execute(stmt); err != nil {
-			return err
-		} else {
-			if resp.GetErrorCode() != graph.ErrorCode_SUCCEEDED {
-				return fmt.Errorf("Response error code: %v, message: %s", resp.GetErrorCode(), resp.GetErrorMsg())
+		if p.deployMode == base.DeployMode_LOCAL {
```

Comment:
The local seems to have the auhtorization problem like SaaS, because local nebula still have the accounts/roles itself, if you are not a god, you update the configs, you may get the error like SaaS.